tests/test_data/examples/spaceman.ldr

```diff
@@ -1,81 +1,81 @@
-1 14 91.175867 10.372700 -119.889286 0.770850 -0.575512 0.273086 0.287269 0.696688 0.657345 -0.568565 -0.428265 0.702369 3820.DAT
-1 2 92.587787 -5.747933 -85.821835 0.212527 -0.647701 -0.731653 0.845369 -0.253662 0.470114 -0.490086 -0.718428 0.493636 3818.DAT
-1 14 87.311647 -17.672686 -100.517502 -0.427542 -0.892429 -0.144145 0.732569 -0.435461 0.523179 -0.529670 0.118085 0.839944 3820.DAT
-1 14 88.371657 -18.724479 -9.496079 0.874087 0.484514 0.034899 -0.413731 0.780187 -0.469186 -0.254555 0.395670 0.882410 3626BPS5.DAT
-1 17 100.000000 0.000000 0.000000 0.874087 0.484514 0.034899 -0.413731 0.780187 -0.469186 -0.254555 0.395670 0.882410 973P90.DAT
-1 1 115.504457 24.965973 12.661438 0.874087 0.484514 0.034899 -0.413731 0.780187 -0.469186 -0.254555 0.395670 0.882410 3815B.DAT
-1 4 121.318629 34.328212 17.409478 0.874087 0.444185 -0.196651 -0.413731 0.468595 -0.780542 -0.254555 0.763622 0.593366 3817B.DAT
-1 2 121.318629 34.328212 17.409478 0.874087 0.453868 -0.173135 -0.413731 0.508803 -0.754948 -0.254555 0.731521 0.632517 3816B.DAT
-1 4 116.987418 0.035522 -0.652969 0.776673 0.341446 0.529334 -0.542924 0.788966 0.287692 -0.319395 -0.510831 0.798147 3819.DAT
-1 14 121.134676 8.687015 -17.797994 0.977567 0.098792 0.186016 -0.042156 0.957077 -0.286753 -0.206361 0.272478 0.939772 3820.DAT
-1 2 90.764811 12.447465 6.983688 0.944943 0.321356 -0.061754 -0.271968 0.666289 -0.694329 -0.181981 0.672896 0.717004 3818.DAT
-1 14 93.003887 31.111210 12.697806 0.928238 0.271891 -0.253870 -0.275777 0.044974 -0.960169 -0.249644 0.961277 0.116728 3820.DAT
-1 14 84.879118 15.766766 90.061790 0.321020 0.630037 0.707107 0.705871 -0.656949 0.264887 0.631421 0.414092 -0.655618 3626BPS5.DAT
-1 17 100.000000 0.000000 100.000000 0.321020 0.630037 0.707107 0.705871 -0.656949 0.264887 0.631421 0.414092 -0.655618 973P90.DAT
-1 1 120.161176 -21.022355 113.250946 0.321020 0.630037 0.707107 0.705871 -0.656949 0.264887 0.631421 0.414092 -0.655618 3815B.DAT
-1 4 127.721617 -28.905738 118.220051 0.321020 0.642282 0.696003 0.705871 -0.652226 0.276312 0.631421 0.402587 -0.662745 3817B.DAT
-1 2 127.721617 -28.905738 118.220051 0.321020 0.240347 0.916067 0.705871 -0.705602 -0.062232 0.631421 0.666603 -0.396166 3816B.DAT
-1 4 109.855590 5.332472 112.784055 0.206738 0.975078 -0.080517 0.809225 -0.124157 0.574230 0.549922 -0.183871 -0.814725 3819.DAT
-1 14 127.983514 1.290639 119.190456 -0.440909 0.575881 -0.688447 0.253981 0.815729 0.519692 0.860867 0.054284 -0.505925 3820.DAT
-1 2 100.224998 -15.843650 93.841418 0.425547 0.755589 0.497990 0.581069 -0.650023 0.489724 0.693735 0.080966 -0.715665 3818.DAT
-1 14 106.885912 -33.625824 98.883886 0.864186 0.492266 -0.104201 0.370929 -0.483322 0.792977 0.339993 -0.723930 -0.600275 3820.DAT
+1 14 -88.775558 -17.237210 -112.364889 -0.879588 -0.467685 -0.087156 -0.435644 0.718217 0.542567 -0.191154 0.515204 -0.835479 3626BPS5.DAT
```

Comment:
have you verified the .ldr is just a reordering of the lines ? should be the same as the spaceman py3 as before

Reply:
I removed the old file and replaced it with py3 version.